sdk/keyvault/keyvault/src/az_keyvault_client.c

```diff
@@ -27,11 +21,79 @@ AZ_NODISCARD az_result az_keyvault_keys_createKey(
     az_span keyname,
     az_span keytype,
     az_keyvault_keys_keys_options * options,
-    az_span * out) {
+    const az_mut_span * const out) {
   (void)client;
   (void)keyname;
   (void)keytype;
   (void)options;
   (void)out;
   return AZ_OK;
 }
+
+AZ_NODISCARD az_result az_keyvault_build_url(
```

Comment:
I assume that his is not intended to be a public function.
Is there a way we can make it clearer what is public and what is not?
Maybe a no-op macro or maybe a macro that expands to 'static' or maybe some method prefix?

Reply:
I highly recommend "static" in front of internal functions like this one that are only defined in .c files, not necessarily for readability, but for technical reasons - static function names are invisible invisible to linker outside of compilation unit, more aggressive optimizations are possible, etc.